packages/@coorpacademy-app-player/locales/en/player.json

```diff
@@ -26,5 +26,7 @@
   "Wrong answer": "Wrong answer",
   "You are out of lives!": "You are out of lives!",
   "Back to question": "Back to question",
-  "Go to question": "Go to question"
+  "Go to question": "Go to question",
+  "Bonus ! Get an extra life by viewing the lesson": "Bonus ! Get an extra life by viewing the lesson",
```

Comment:
`Bonus ! Get an extra life by viewing the lesson`
-->
`Bonus! Get an extra life by viewing the lesson`

Vu la gueule de la clé de traduction, et la probabilité que ça change à un moment, mettez peut-être une clé avec un nom plus générique, du style `extra_life_suggestion` ? C'est une idée mais qui va à l'encontre de ce qui a été fait dans l'app-player (mais je sais pas pourquoi on fait pas des clés de trads comme on fait dans le MOOC honnêtement).

Reply:
yep ca me va !